### PR TITLE
Performance Improvements

### PR DESCRIPTION
**Pull request by alexrisch**
Removed useClient

Moved entify into query
Updated QueryKeys
Made GroupAvatar not request on conversation list screen

### CODE REVIEW

File: queries/useGroupMessages.ts [hunk 1]
```diff
@@ -13,16 +13,8 @@ export const useGroupMessages = (account: string, topic: string) => {
         return;
```
Comment (nmalzieu) [inline queries/useGroupMessages.ts:13]: shouldn't we return 
```
{
  byId: {},
  ids: []
}
```

here?

File: components/ConversationList/GroupConversationItem.tsx [hunk 2]
```diff
@@ -38,9 +38,13 @@ export const GroupConversationItem: FC<GroupConversationItemProps> = ({
...
     refetchOnMount: false,
+    staleTime: Infinity,
+    gcTime: Infinity,
   });
   const { data: groupPhoto } = useGroupPhotoQuery(userAddress, topic, {
     refetchOnMount: false,
+    staleTime: Infinity,
+    gcTime: Infinity,
   });
   const { blockGroup } = useGroupConsent(topic);
```
Comment (nmalzieu) [inline components/ConversationList/GroupConversationItem.tsx:49]: `useGroupConsent` will call many other queries and needs to be prevented to query the bridge also?

**Review by nmalzieu — CHANGES_REQUESTED** [review]
Thanks, looks cleaner!

Just did a quick code review without testing as I think there's a bit more to do

Once it's done I'll give it a go, adding logs when querying the bridge, to see the improvements and if there are other places that need work!

File: components/ConversationList/GroupConversationItem.tsx (hunk 2 above)
Comment (alexrisch) [reply components/ConversationList/GroupConversationItem.tsx:49]: Good catch! I'll update for that too

Comment (alexrisch) [reply components/ConversationList/GroupConversationItem.tsx]: Updated!
Verified it's not being called on the conversation list after initial load any more

**Comment by nmalzieu** [comment]
I have successfully managed to break the app again on a big account with many groups.

By commenting some queries I can unblock it so I still think it's linked to react queries… I'm still unsure why it just locks the app, it's like the JS thread is blocked.

Small investigation:

- `useGroupNameQuery` used in the group list item calls `useGroupQuery` which calls `useGroupsQuery` : we should pass along queryOptions or it might trigger calls 
- same for `useGroupPhotoQuery` 
- `useGroupConsent` calls `useGroupQuery` but also `useGroupCreator` which also calls `useGroupQuery` they should probably also get the change

**Comment by alexrisch** [comment]
Closing this out in favor of the other branch

**Pull request closed without merging**